team.html

```diff
@@ -1269,8 +1273,41 @@ <h1>Developers</h1>
 						<span>Full Stack Developer</span>
 					</div>
 				</div>
-			</div>
-			</div>
+
+				<div class="col-md-3 col-sm-6 speaker-column">
+					<div class="speaker">
+						<div class="image-holder">
+							<img class="background-image" alt="Vibhav Singh" src="img/VaibhavSingh.jpg">
+							<div class="hover-state text-center preserve3d">
+								<div class="social-links vertical-align">
+									<a href="https://github.com/vaibhavsingh97"><i class="icon fa fa-github"></i></a>
+									<a href="https://www.facebook.com/profile.php?id=100002720353851&ref=br_rs"><i class="icon social_facebook"></i></a>
+									<a href="https://twitter.com/vaibhavsingh97"><i class="icon social_twitter"></i></a>
```

Comment:
Can you please add website (https://vaibhavsingh97.me/) and linkedin (https://www.linkedin.com/in/vaibhav-singh/) 😅